### PR TITLE
feat(DCS): dcs supports ssl updated

### PR DESCRIPTION
**Pull request by 346010762**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/dcs/ TESTARGS='-run TestAccDcsInstances'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dcs/ -v -run TestAccDcsInstances -timeout 360m -parallel 4
=== RUN   TestAccDcsInstances_basic
=== PAUSE TestAccDcsInstances_basic
=== RUN   TestAccDcsInstances_ha_change_capacity
=== PAUSE TestAccDcsInstances_ha_change_capacity
=== RUN   TestAccDcsInstances_ha_expand_replica
=== PAUSE TestAccDcsInstances_ha_expand_replica
=== RUN   TestAccDcsInstances_ha_to_proxy
=== PAUSE TestAccDcsInstances_ha_to_proxy
=== RUN   TestAccDcsInstances_rw_change_capacity
=== PAUSE TestAccDcsInstances_rw_change_capacity
=== RUN   TestAccDcsInstances_rw_expand_replica
=== PAUSE TestAccDcsInstances_rw_expand_replica
=== RUN   TestAccDcsInstances_rw_to_proxy
=== PAUSE TestAccDcsInstances_rw_to_proxy
=== RUN   TestAccDcsInstances_proxy_change_capacity
=== PAUSE TestAccDcsInstances_proxy_change_capacity
=== RUN   TestAccDcsInstances_proxy_to_ha
=== PAUSE TestAccDcsInstances_proxy_to_ha
=== RUN   TestAccDcsInstances_proxy_to_rw
=== PAUSE TestAccDcsInstances_proxy_to_rw
=== RUN   TestAccDcsInstances_cluster_change_capacity
=== PAUSE TestAccDcsInstances_cluster_change_capacity
=== RUN   TestAccDcsInstances_cluster_expand_replica
=== PAUSE TestAccDcsInstances_cluster_expand_replica
=== RUN   TestAccDcsInstances_withEpsId
=== PAUSE TestAccDcsInstances_withEpsId
=== RUN   TestAccDcsInstances_whitelists
=== PAUSE TestAccDcsInstances_whitelists
=== RUN   TestAccDcsInstances_tiny
=== PAUSE TestAccDcsInstances_tiny
=== RUN   TestAccDcsInstances_single
=== PAUSE TestAccDcsInstances_single
=== RUN   TestAccDcsInstances_prePaid
=== PAUSE TestAccDcsInstances_prePaid
=== RUN   TestAccDcsInstances_ssl
=== PAUSE TestAccDcsInstances_ssl
=== CONT  TestAccDcsInstances_basic
=== CONT  TestAccDcsInstances_proxy_to_rw
=== CONT  TestAccDcsInstances_rw_expand_replica
=== CONT  TestAccDcsInstances_rw_change_capacity
--- PASS: TestAccDcsInstances_basic (211.95s)
=== CONT  TestAccDcsInstances_ssl
--- PASS: TestAccDcsInstances_rw_expand_replica (271.76s)
=== CONT  TestAccDcsInstances_prePaid
    acceptance.go:612: This environment does not support prepaid tests
--- SKIP: TestAccDcsInstances_prePaid (0.01s)
=== CONT  TestAccDcsInstances_single
--- PASS: TestAccDcsInstances_rw_change_capacity (308.10s)
=== CONT  TestAccDcsInstances_tiny
--- PASS: TestAccDcsInstances_ssl (176.87s)
=== CONT  TestAccDcsInstances_whitelists
--- PASS: TestAccDcsInstances_single (131.66s)
=== CONT  TestAccDcsInstances_withEpsId
    acceptance.go:425: The environment variables does not support Migrate Enterprise Project ID for acc tests
--- SKIP: TestAccDcsInstances_withEpsId (0.01s)
=== CONT  TestAccDcsInstances_cluster_expand_replica
--- PASS: TestAccDcsInstances_tiny (133.49s)
=== CONT  TestAccDcsInstances_cluster_change_capacity
--- PASS: TestAccDcsInstances_whitelists (180.13s)
=== CONT  TestAccDcsInstances_ha_expand_replica
--- PASS: TestAccDcsInstances_proxy_to_rw (646.78s)
=== CONT  TestAccDcsInstances_ha_to_proxy
--- PASS: TestAccDcsInstances_ha_expand_replica (253.38s)
=== CONT  TestAccDcsInstances_proxy_change_capacity
--- PASS: TestAccDcsInstances_cluster_expand_replica (473.71s)
=== CONT  TestAccDcsInstances_proxy_to_ha
--- PASS: TestAccDcsInstances_ha_to_proxy (552.39s)
=== CONT  TestAccDcsInstances_rw_to_proxy
--- PASS: TestAccDcsInstances_proxy_to_ha (667.89s)
=== CONT  TestAccDcsInstances_ha_change_capacity
--- PASS: TestAccDcsInstances_cluster_change_capacity (1114.05s)
--- PASS: TestAccDcsInstances_rw_to_proxy (565.32s)
--- PASS: TestAccDcsInstances_ha_change_capacity (318.15s)
--- PASS: TestAccDcsInstances_proxy_change_capacity (1737.73s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dcs       2560.092s

```

### CODE REVIEW

File: huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go [hunk 1]
```diff
@@ -703,6 +725,20 @@ func resourceDcsInstancesCreate(ctx context.Context, d *schema.ResourceData, met
 		}
 	}
 
+	sslEnabled := d.Get("ssl_enable").(bool)
+	if sslEnabled {
```
Comment (houpeng80) [inline huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go]:
```suggestion
	sslEnabled := d.Get("ssl_enable").(bool)
	if sslEnabled := d.Get("ssl_enable").(bool); sslEnabled  {
```

Comment (346010762) [reply huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go]: done

**Pull request merged**